### PR TITLE
Support activating multiple sticky features

### PR DESCRIPTION
**Pull request by mpelzsherman**

### CODE REVIEW

File: test/spec/pinball_wizard_spec.coffee [hunk 1]
```diff
@@ -267,6 +267,23 @@ define ['pinball_wizard'], (pinball) ->
       urlParam = '?foo=bar&pinball=a,b&bar'
       expect(pinball._urlValues(urlParam)).toEqual(['a','b'])
 
+  describe '#activatePermanently', ->
```
Comment (c0) [inline test/spec/pinball_wizard_spec.coffee]: Can we combine this describe block with the next? I have a typo on the method name :-/

File: src/pinball_wizard.coffee [hunk 2]
```diff
@@ -135,8 +135,9 @@ define ->
     JSON.parse(storage.getItem('pinball_wizard') or "[]") or []
 
   activatePermanently = (name) ->
-    appendPermanent(name)
-    activate(name, 'permanent')
+    for n in name.split(',')
```
Comment (c0) [inline src/pinball_wizard.coffee]: What do you think about using coffeescript splat? http://coffeescript.org/#splats

Comment (mpelzsherman) [inline src/pinball_wizard.coffee]: Sorry, I'm not sure what you're suggesting.

Comment (c0) [inline src/pinball_wizard.coffee]:
```coffee
activatePermanently = (names...) ->
  for n in names
    appendPermanent(n)
    activate n, 'permanent'
```

Usage:
`pinball.activatePermanently('feature_a','feature_b')`

Comment (c0) [inline src/pinball_wizard.coffee]: Yeah, it does take more characters (2), but it's odd to pass in a comma delimited string when working with the lib directly.

Comment (mpelzsherman) [inline src/pinball_wizard.coffee]: OK, splat it is!

**Comment by c0** [comment]
:+1:

**Pull request merged**